.travis.yml

```diff
@@ -9,4 +9,4 @@ branches:
 jobs:
   include:
     - stage: "Lint" # naming the Tests stage
-      script: npm run lint
+      script: npm run lint --no-fix
```

Comment:
I meant here: https://github.com/viniciuskneves/vue-eleicoes-2018/blob/master/package.json#L26

Reply:
Okay, I went ahead and made those changes. I made a new command called `lint:ci` so the original lint command could be used in other situations.

Reply:
If this isn't a good resolution let me know and I'll just modify the original lint task 😄